client/templates/home/home.js

```diff
@@ -62,11 +62,7 @@ Template.gameList.helpers({
 });
 
 Template.gameList.events({
-    'click .button-small': function(e) {
-        Meteor.call('joinGame', $(e.currentTarget).attr('id'), function(error, results) {
-            if (error) {
-                console.log(error);
-            }
-        });
+    'click #joinGame': function(e) {
+        Router.go("/upload/" + $(e.target).attr("gameId"));
```

Comment:
Our project is getting to big. We need svn
On Nov 9, 2015 8:27 PM, "rsk7" <notifications@github.com> wrote:

> In client/templates/home/home.js
> <https://github.com/brianchitester/facequalm/pull/57#discussion_r44359735>
> :
>
> > @@ -62,11 +62,7 @@ Template.gameList.helpers({
> >  });
> >
> >  Template.gameList.events({
> > -    'click .button-small': function(e) {
> > -        Meteor.call('joinGame', $(e.currentTarget).attr('id'), function(error, results) {
> > -            if (error) {
> > -                console.log(error);
> > -            }
> > -        });
> > +    'click #joinGame': function(e) {
> > +        Router.go("/upload/" + $(e.target).attr("gameId"));
>
> after linking to that file, i realized that the server side refactor I did
> of joinGame isn't right because of line 170. check out pull #58
> <https://github.com/brianchitester/facequalm/pull/58>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brianchitester/facequalm/pull/57/files#r44359735>.
>